apps/smart-contracts/core/test/unit/WithdrawHook.test.ts

```diff
@@ -351,6 +351,22 @@ describe('=> WithdrawHook', () => {
         ).revertedWith('Global withdraw limit exceeded')
       })
     })
+
+    it('reverts if global cap already exceeded', async () => {
+      await withdrawHook.connect(deployer).setGlobalWithdrawLimitPerPeriod(TEST_USER_WITHDRAW_LIMIT)
+      await withdrawHook
+        .connect(fakeCollateral.wallet)
+        .hook(user.address, user.address, TEST_USER_WITHDRAW_LIMIT, TEST_USER_WITHDRAW_LIMIT)
+      await withdrawHook
+        .connect(deployer)
+        .setGlobalWithdrawLimitPerPeriod(TEST_USER_WITHDRAW_LIMIT.sub(1))
+
+      const tx = withdrawHook
+        .connect(fakeCollateral.wallet)
+        .hook(user.address, user.address, TEST_USER_WITHDRAW_LIMIT, TEST_USER_WITHDRAW_LIMIT)
```

Comment:
```suggestion
        .hook(user.address, user.address, 0, 0)
```
should work, since the limit is already exceeded